css/style.css

```diff
@@ -0,0 +1,6 @@
+#image {
+    max-width: 100%;
+    display: block;
+    height: auto;
+    margin: 0 auto;
```

Comment:
Please add only two spaces before writing the CSS properties. That way, your code will look more professional. 😎😎😎